Resources/Prototypes/_RMC14/Entities/Clothing/Ears/distress_headsets.yml

```
@@ -46,3 +46,15 @@
       key_slots:
       - CMEncryptionKeyCommon
       - RMCEncryptionKeyLiaison
+
+# For Dutchs Dozen/Freelancer
+- type: entity
+  parent: CMHeadset
+  id: RMCHeadsetDistressDutch
```

This is easier for now while we come up with a name
```suggestion
  id: RMCHeadsetDistressFreelancer
```